neuro_evolution_ctrnn/tools/env_handler.py

```diff
@@ -78,45 +74,85 @@ def make_env(self, env_id: str):
 
-
         if str(env_id).startswith("BipedalWalker"):
-            logging.info("wrapping env in Box2DWalkerWrapper")
+            logging.debug("wrapping env in Box2DWalkerWrapper")
             env = Box2DWalkerWrapper(env)
 
         if self.config.novelty:
             if self.config.novelty.behavior_source in ['observation', 'action', 'state']:
-                logging.info("wrapping env in BehaviorWrapper")
+                logging.debug("wrapping env in BehaviorWrapper")
                 env = BehaviorWrapper(env, self.config.novelty.behavior_source,
                                       self.config.novelty.behavioral_interval,
                                       self.config.novelty.behavioral_max_length)
 
         if self.config.max_steps_per_run:
-            logging.info("wrapping env in MaxStepWrapper")
+            logging.debug("wrapping env in MaxStepWrapper")
             env = MaxStepWrapper(env, max_steps=self.config.max_steps_per_run, penalty=self.config.max_steps_penalty)
 
         return env
 
 
-class ProcEnvWrapper(Wrapper):
-
-    def __init__(self, env):
-        super(ProcEnvWrapper, self).__init__(env)
-        self.screen_size = 32
+class ProcEnvHandler(gym.Env):
+    """
+    This Wrapper scales to observation to values between 0 and 1.
+    Additionally it implements a seed method because for reasons unknown it not implemented upstream
+    """
+
+    def __init__(self, env_id, render):
+        # todo: maybe add env specific configuration, but only after issue #20 has been implemented
+        self.env_id = env_id
+        self.render_mode = None
+        if render:
+            self.render_mode = 'rgb_array'
+        super(ProcEnvHandler, self).__init__()
+        self._env = self._make_inner_env(start_level=0)
+        self.spec = copy.deepcopy(self._env.spec)  # deep copy to avoid references to inner gym
+        self.action_space = self._env.action_space  # use reference, so action_space.seed() works as expected
         self.obs_dtype = np.float16
+        self.input_high = 255
+        self.current_level = 0
+        assert self.input_high == self._env.observation_space.high.min(), "unexpected bounds for input space"
+        assert self.input_high == self._env.observation_space.high.max(), "unexpected bounds for input space"
+        assert 0 == self._env.observation_space.low.min(), "unexpected bounds for input space"
+        assert 0 == self._env.observation_space.low.max(), "unexpected bounds for input space"
         self.observation_space = Box(low=0, high=1,
-                                     shape=(self.screen_size, self.screen_size, 3),
+                                     shape=self._env.observation_space.shape,
                                      dtype=self.obs_dtype)
 
+    def _make_inner_env(self, start_level):
+        self.current_level = start_level
+        env = gym.make(self.env_id,
```

Comment:
```suggestion
    def _make_inner_env(self, start_level):
        self.current_level = start_level
        # parameters are explained here: https://github.com/openai/procgen#environment-options
        # num_levels=1 enforces that start_level is the first level
        # otherwise levels would be selected randomly between start_level and start_level+num_levels
        env = gym.make(self.env_id,
```